### ISSUE TITLE
`mask_loss` is missing on softmax.json config

### ISSUE THREAD

**Issue by bezero** [selimsef/xview2_solution#3]
Hi, @selimsef 
I want to replicate your training for classification. From what I understood, `x_softmax.json` files are used to run `train.py`. However, `train.py` is expecting `mask_loss` configuration to run but it is missing in all softmax config files. Am I missing something here?

**Comment by selimsef** [comment]
Hi, you are right. This was totally a wrong training script from first experiments where I used binary masks for each class. Fixed in master
